### PR TITLE
Add a new profession: Para-Amputee

### PR DESCRIPTION
**Pull request by MNG-cataclysm**
#### Summary
Content "Add a new profession, Para-Amputee"

#### Purpose of change
Adds a new profession similar to the one proposed in #68274.

#### Describe the solution
Adds a new profession, the Para-Amputee. This is someone who has lost both of their legs at some point. They start with a wheelchair, and have a bonus to vehicles and athletics to effectively use it.
You can select this profession normally, and I have incorporated it into several challenge scenarios where it would make sense, like Abandoned, Golden Years, Mutant, and Lab Patient. For the more scientific scenarios, I added a variant which comes with lab-themed equipment and a bonus to skills carried over from the Volunteer Mutant.

#### Describe alternatives you've considered
Not doing this. 

#### Testing
Everything works great.

#### Additional context
If this is merged, I plan to add some sort of replacement item for your legs and feet, like a prosthetic to help with tight spaces or places your wheelchair couldn't go. Additionally, if anybody can think of one, I'd like to rename it to something more professional, but I couldn't find a good medical term for someone who has lost both of their legs.

### CODE REVIEW

**Comment by EvanBalster** [comment]
A bit of inspiration courtesy of AbleGamers org.  Maybe you can get some ideas from Cherry's talk?

https://www.youtube.com/watch?v=Vb39BFs1UK0

https://newmobility.com/video-game-representation/

**Comment by MNG-cataclysm** [comment]
> A bit of inspiration courtesy of AbleGamers org. Maybe you can get some ideas from Cherry's talk?
> 
> A key thing to note is people wouldn't _necessarily_ pick this profession for the difficulty factor. Some folks with disabilities jump at the chance to play a character whose life experiences parallel their own. This could be a good pretext for e.g. a special proficiency to reflect skills acquired pre-cataclysm.
> 
> https://www.youtube.com/watch?v=Vb39BFs1UK0
> 
> https://newmobility.com/video-game-representation/

Thank you, I'll definitely need to check this out.

**Comment by Maleclypse** [comment]
So I'm going to pull this to my local and test it with gastropod mutagen to see if that eventually grows in the gastropod foot. If someone wants to test that and share pictures of it working or state that it doesn't please do so as I am not sure if I'll get to that tonight or in the next week.

File: data/json/mutations/mutations.json [hunk 1]
```diff
@@ -308,6 +308,32 @@
...
     "movecost_flatground_modifier": 0.85
   },
+  {
+    "type": "mutation",
+    "id": "NO_LEGS",
+    "name": { "str": "No Legs" },
+    "points": 0,
+    "valid": false,
+    "purifiable": false,
+    "description": "Wether by a serious accident, birth, defect, or other means, you have lost your legs.  You can crawl around, but will likely need a wheelchair or other means of locomotion to get somewhere far.",
```
Comment (TheShadowFerret) [inline data/json/mutations/mutations.json:318]:
```suggestion
    "description": "Whether by a serious accident, birth, defect, or other means, you have lost your legs.  You can crawl around, but will likely need a wheelchair or other means of locomotion to get somewhere far.",
```

**Comment by MNG-cataclysm** [comment]
I've noticed a problem with missing limbs throwing errors, I've started working on a fix for that.

**Comment by MNG-cataclysm** [comment]
Problem solved.

**Pull request merged**